plugin/src/main/java/io/jenkins/plugins/casc/impl/secrets/VaultSecretSource.java

```diff
@@ -82,7 +87,9 @@ public VaultSecretSource() {
                     LOGGER.log(Level.FINE, "Login to Vault using U/P successful");
                 }
                 config.token(token).build();
-                secrets = vault.logical().read(vaultPth).getData();
+                for(String vaultPath : vaultPaths) {
+                    secrets.putAll(vault.logical().read(vaultPath).getData());
```

Comment:
Hmm, should something happen (logs or fail) when keys are overwritten by another secret path?

Reply:
Good point. I added a warning now if an already existing key from another paths is overwritten.